### PR TITLE
Updates to use new values schema for Helm Chart

### PR DESCRIPTION
**Pull request by rukykf**

### CODE REVIEW

File: examples/de-aws-eks/jwt-public-key.pem [hunk 1]
```diff
@@ -0,0 +1,9 @@
+-----BEGIN PUBLIC KEY-----
```
Comment (lazyoldbear) [inline examples/de-aws-eks/jwt-public-key.pem:1]: Should we, perhaps, generate it and `.gitignore`? I feel so bad about publishing keys. Especially only part of the pair.

File: examples/de-aws-eks/README.md [hunk 2]
```diff
@@ -38,13 +38,13 @@ You will need an AWS account, and a [profile set for it](https://docs.aws.amazon
 You can use the AWS CLI to set this up by running `aws configure --profile document-engine-example`. After running this command, the configuration wizard will guide you through setting up the profile. 
 
 Or you can manually edit the `~/.aws/*` configuration files. 
-This entails something like this in `~/.aws/config`:
+This entails something like this in `~/.aws/credentials`:
```
Comment (lazyoldbear) [inline examples/de-aws-eks/README.md:41]: Not both?

File: examples/de-aws-eks/README.md [hunk 3]
```diff
@@ -55,7 +55,7 @@ Prepare Terraform environment by setting the AWS profile and (optionally) region
 
 ```shell
 export TF_VAR_aws_profile_name="document-engine-example"
-export TF_VAR_aws_region="eu-north-1"
+export TF_VAR_aws_region="eu-north-1" # Remove or change the default in `terraform.tfvars` file if setting this variable
```
Comment (lazyoldbear) [inline examples/de-aws-eks/README.md:58]: What's wrong with Stockholm? 
Cinnamon buns are all right there.

**Review by lazyoldbear — APPROVED** [review]

File: examples/de-aws-eks/README.md (hunk 2 above)
Comment (rukykf) [reply examples/de-aws-eks/README.md:41]: The specific example in this readme is from `~/.aws/credentials`

File: examples/de-aws-eks/README.md (hunk 3 above)
Comment (rukykf) [reply examples/de-aws-eks/README.md:58]: Nothing wrong with it but the `.tfvars` file always overrides the environment variables so if someone wants to run the example in a different region this is something to keep in mind.

**Pull request merged**